app/components/OpenLayers.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { useEffect } from 'react';
+
+import TileLayer from 'ol/layer/Tile';
+import { OSM } from 'ol/source';
+import { Map, View } from 'ol';
+
+import 'ol/ol.css';
+import styles from './OpenLayers.module.css';
+
+interface Props {
+    id: string
+}
+
+function OpenLayers({ id }: Props) {
+    useEffect(() => {
+        const osmLayer = new TileLayer({
+            preload: Infinity,
+            source: new OSM(),
+        })
+
+        const map = new Map({
+            target: id,
+            layers: [ osmLayer ],
+            view: new View({
+                center: [0, 0],
+                zoom: 0,
```

Comment:
Wäre es nicht sinnvoller, dass initialer Zoom, wie auch Startposition in die Konfiguration rausgehen, damit diese einfach angepasst werden können?

Reply:
Im Moment sehe ich das eigentlich noch nicht als notwendig an. Auch im nächsten Schritt, wo die Ansicht automatisch angepasst werden soll, ist das nicht notwendig.